modules/page/html/render/channel.js

```diff
@@ -70,6 +76,20 @@ exports.create = function (api) {
         return `#${api.channel.sync.normalize(link.slice(1))}`
       }
     }
+
+    function noVisibleNewPostsWarning() {
+      var content =
+        h('div', {classList: 'NotFollowingAnyoneWarning'}, h('section -notFollowingAnyoneWarning', [
+          h('h1', i18n('You are not Following Anyone')),
+          h('p', i18n('You may not be able to see new channel content until you follow some users or pubs.')),
+          h('p', [i18n("For help, see the 'Getting Started' guide at "),
+            h('a', {href: 'https://www.scuttlebutt.nz/'}, 'https://www.scuttlebutt.nz/')]
```

Comment:
if we want to link directly to the "Getting Started" page, it's now available at: <https://www.scuttlebutt.nz/getting-started.html>